README.md

```diff
@@ -76,6 +77,13 @@ Visit our [website](http://flyve.org/flyve-mdm-android-inventory-agent/) for mor
 |:---:|:---:|
 | [![Build Status](https://circleci.com/gh/flyve-mdm/flyve-mdm-android-inventory-agent/tree/master.svg?style=svg)](https://circleci.com/gh/flyve-mdm/flyve-mdm-android-inventory-agent/tree/master) | [![Build Status](https://circleci.com/gh/flyve-mdm/flyve-mdm-android-inventory-agent/tree/develop.svg?style=svg)](https://circleci.com/gh/flyve-mdm/flyve-mdm-android-inventory-agent/tree/develop) |
 
+## Compatibility Matrix
+
+|GLPI|9.1.1|9.1.2|9.1.3|9.1.4|9.1.5|9.1.6|9.2.0|
```

Comment:
Please, use MINOR version only, no PATCH.
Example: |GLPI|9.1|9.2|
Because no changes from PATCH to PATCH (bugfixes), only in MINOR new features are added.